package.json

```
@@ -11,8 +11,10 @@
 		"url": "https://sindresorhus.com"
 	},
 	"engines": {
-		"node": ">=10"
+		"node": ">=12.2"
```

```suggestion
		"node": ">=12.20"
```

I'm going with 12.20 generally, since I want to use the `node:` protocol too.